### PR TITLE
Network Layer Update

### PR DESCRIPTION
**Pull request by MohamedSaiko**
Requesting code review for the network layer.

### CODE REVIEW

File: iStore/Constants.swift [hunk 1]
```diff
@@ -10,11 +10,12 @@ import Foundation
 // MARK: NavigationBar Titles
 
 let iStore = "iStore"
-let category = "Category"
+let categories = "Categories"
```
Comment (akabel92) [inline iStore/Constants.swift:13]: Why did you change this here? Can't see the usage for it

Comment (MohamedSaiko) [reply iStore/Constants.swift:13]: Agree

Comment (MohamedSaiko) [reply iStore/Constants.swift:13]: better naming but I should have added it to the categories branch not here.

**Review by akabel92 — APPROVED** [review]
